asset-transfer-basic/chaincode-external-nano/README.md

```diff
@@ -0,0 +1,78 @@
+# Asset-Transfer-Basic as an external service - Nano
+
+This sample complements the [Test Network Nano sample](../../test-network-nano-bash) demonstrating how you can run the chaincode as an external service without using any container at all.
+
+## Setting up the external builder and launcher
+
+Fabric's external builders and launchers is an advanced feature that typically requires custom packaging of the peer image so that it contains all the tools your builder and launcher require. This sample uses very simple (and crude) shell scripts that can be run directly by the default Fabric peer.
+
+Open the `config/core.yaml` file at the top of the `fabric-samples` directory. If you do not have this file, follow the instructions to [Install the Samples, Binaries and Docker Images](https://hyperledger-fabric.readthedocs.io/en/latest/install.html) to download the Fabric binaries and configuration files alongside the Fabric samples.
+
+Modify the `externalBuilders` field in the `core.yaml` file to add the external sample builder which comes with this sample so that the configuration looks something like the following:
+```
+externalBuilders:
+    - name: external-sample-builder
```

Comment:
_cf_ comment re the current core.yaml being pre-configured

Reply:
You still need to modify the config file though because the path specified by default is only valid within the container. I don't think instructing people to install the ccaasbuilder in /opt/hyperledger/ccaas_builder on their machine is really an option.

Reply:
good point!